packages/manager/apps/telecom/src/app/telecom/pack/xdsl/pack-xdsl.html

```diff
@@ -24,38 +24,28 @@
                 </div>
             </div>
         </header>
-
-        <nav role="navigation">
-            <responsive-tabs>
-                <responsive-tab
-                    data-state="telecom.packs.pack.xdsl"
-                    data-dropdown-title="{{ 'xdsl_tabs_line' | translate }}"
-                >
-                    <span data-translate="xdsl_tabs_line"></span>
-                </responsive-tab>
-                <responsive-tab
-                    data-state="telecom.packs.pack.xdsl.modem"
-                    disabled="$ctrl.disabledModem"
-                    data-ng-if="$ctrl.isModemTabAvailable()"
-                    data-dropdown-title="{{ 'xdsl_tabs_modem' | translate }}"
-                >
-                    <span data-translate="xdsl_tabs_modem"></span>
-                </responsive-tab>
-                <responsive-tab-more>
-                    <span
-                        data-translate-attr="{title: 'common_tabs_more'}"
-                        class="fa fa-plus"
-                        aria-hidden="true"
-                    ></span>
-                </responsive-tab-more>
-                <responsive-tab
-                    data-state="telecom.packs.pack.xdsl.tasks"
-                    data-dropdown-title="{{ 'xdsl_tabs_tasks' | translate }}"
-                >
-                    <span data-translate="xdsl_tabs_tasks"></span>
-                </responsive-tab>
-            </responsive-tabs>
-        </nav>
+        <oui-header-tabs>
+            <oui-header-tabs-item
+                href="{{ $ctrl.lineLink }}"
```

Comment:
data- please

Reply:
Not mandatory as it is a custom component